semantic_conventions/trace/cloudevents.yaml

```diff
@@ -0,0 +1,40 @@
+groups:
+  - id: cloudevents
+    prefix: cloudevents
+    brief: >
+        This document defines attributes for CloudEvents.
+        CloudEvents is a specification on how to define event data in a standard way.
+        These attributes can be attached to spans when performing operations with CloudEvents, regardless of the protocol being used.
+    attributes:
+      - id: event_id
+        type: string
+        brief: >
+          The [event_id](https://github.com/cloudevents/spec/blob/master/spec.md#id) identifies the event within the scope of a producer.
+        examples: ['123e4567-e89b-12d3-a456-426614174000', 'producer-1']
+      - id: source
+        type: string
+        brief: >
+          The [source](https://github.com/cloudevents/spec/blob/master/spec.md#source-1) identifies the context in which an event happened.
+        examples: ['https://github.com/cloudevents', '/cloudevents/spec/pull/123', 'my-service' ]
+      - id: spec_version
+        type: string
+        brief: >
+          The [version of the CloudEvents specification](https://github.com/cloudevents/spec/blob/master/spec.md#specversion) which the event uses.
+        examples: '1.0'
+      - id: event_type
+        type: string
+        brief: >
+          The [event_type](https://github.com/cloudevents/spec/blob/master/spec.md#type) contains a value describing the type of event related to the originating occurrence.
+        examples: ['com.github.pull_request.opened', 'com.example.object.deleted.v2']
+      - id: data_content_type
```

Comment:
Is this actually of interest?

Reply:
Good point. For a consumer it's likely necessary to know (unless they can't tell otherwise) but for tracing it's probably not really valuable.

Reply:
I guess it depends? I mean if I'm investigating an issue where something happened while reading an event, having the content type of the 'data' inside the event might be useful?

Reply:
Ok I removed it for now. If it comes up to be relevant we can always add it later.